src/databricks/labs/ucx/install.py

```diff
@@ -227,7 +227,7 @@ def run_workflow(self, step: str):
         logger.debug(f"starting {step} job: {self._ws.config.host}#job/{job_id}")
         job_run_waiter = self._ws.jobs.run_now(job_id)
         try:
-            job_run_waiter.result()
+            job_run_waiter.result(timeout=datetime.timedelta(days=1))
```

Comment:
Make timeout a parameter of run_workflow(), setting 1 day timeout here is a hidden disaster